datahub-web-react/src/app/entity/shared/containers/profile/sidebar/Ownership/EditOwnersModal.tsx

```diff
@@ -78,10 +78,25 @@ export const EditOwnersModal = ({
     const renderSearchResult = (entity: Entity) => {
         const avatarUrl =
             (entity.type === EntityType.CorpUser && (entity as CorpUser).editableProperties?.pictureLink) || undefined;
+        const isCorpUser = (entity.type as EntityType) === EntityType.CorpUser;
+        const displayCorpUserDepartmentName =
```

Comment:
By the name of this variable, it would sound like it were a boolean. 

I would suggest

corpUserDepartmentName
corpUserName

and just leave them as null or undefined if there is no value.

Reply:
Let's call displayCorpUserUserName corpUserId, that's generally what we refer to the "username" field as (A unique id for the user, could be email or beyond)

Reply:
Addressed